### ISSUE TITLE
Merge Widgets

### ISSUE THREAD

**Issue by tpnsolutions** [freescout-helpdesk/freescout#1509]
When you purchase multiple modules that have widgets they all come with individual embeddable widgets.

While being able to embed them individually makes sense if you don't intend to use them all, if like me you DO want to make use of more then one It'd be nice to have functionality merged into a widget perhaps allowing us to choose which features to include.

For instance I have "Knowledge Base", "End User Portal" and "Chat" modules.

When I'm available for chat it would be nice to allow people to search the "Knowledge Base" or open a "Chat" but when I'm offline it'd be nice to let them search the "Knowledge Base" or submit a ticket from the widget.

On a segway it'd also be nice if the "Knowledge Base" could link to the "End User Portal" in the upper right menu in the same type of merged functionality making things more useful.

Bottom line, while working individually I believe modules should equally compliment each other a bit more.

*** Recently moved away from FreshDesk and never looking back, kudos! ***
